### PR TITLE
✅ Improve test coverage for core/Heartbeat.py (#121)

### PR DESCRIPTION
**Pull request by gillesigot**
### Changes

-   `adds`: new test for core object Heartbeat

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

**Comment by Arthurdw** [comment]
Could you add the following snipped to the appropriate files which are missing it.
```py
# Copyright Pincer 2021-Present
# Full MIT License can be found in `LICENSE` at the project root.
```

File: tests/utils/utils.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from contextlib import contextmanager
+
+
+@contextmanager
```
Comment (Sigmanificient) [inline tests/utils/utils.py:4]: shouldn't that be within the tests part?

Comment (gillesigot) [reply tests/utils/utils.py]: Indeed, I moved it to `tests`. I didn't realize it could bring confusion with the actual testing of the `utils` package.

**Review by Sigmanificient — APPROVED** [review]

**Review by Arthurdw — APPROVED** [review]
@Sigmanificient I think we are able to merge this without creating conflicts in #155 .

**Comment by gillesigot** [comment]
@Sigmanificient Hey, I'm doing the Hacktoberfest, but seems this PR is not counted. Could you please add the tag  `hacktoberfest-accepted` to this one?

**Comment by Sigmanificient** [comment]
We hope your pr will get accepted!

**Pull request merged**